src/main/java/ticTacToe/ManagerTicTacToe.java

```diff
@@ -48,14 +48,62 @@ public DisplayBoard getDisplayBoard() {
         return displayBoard;
     }
 
+    public void setPlayer(Player player) {
+        this.player = player;
+    }
+
+    public void setArtificialPlayer(ArtificialPlayer artificialPlayer) {
+        this.artificialPlayer = artificialPlayer;
+    }
 
+    public void playGame(){
+        displayBoard.showBoardWhenPlay();
+        while (true){
+            System.out.println("Your move");
+            takeMoveFromPlayer();
+            displayBoard.showBoardWhenPlay();
+                if(!(" ").equals(checkIfGameIsOver.checkIsTheWinner(this.board.getMyBoard()))){
+                    String isWinner = checkIfGameIsOver.checkIsTheWinner(this.board.getMyBoard());
+                    System.out.println("The winner is " + isWinner);
+                    return;
+                }
+                if(checkIfGameIsOver.checkAreAllFieldsTaken(this.board.getMyBoard())){
+                    System.out.println("Game over.");
+                    return;
+                }
+            System.out.println("Artificial player ");
+            takeMovesFromArtificialPlayer();
+            displayBoard.showBoardWhenPlay();
+                if(!(" ").equals(checkIfGameIsOver.checkIsTheWinner(this.board.getMyBoard()))){
+                    String isWinner = checkIfGameIsOver.checkIsTheWinner(this.board.getMyBoard());
+                    System.out.println("The winner is " + isWinner);
+                    return;
+                }
+                if(checkIfGameIsOver.checkAreAllFieldsTaken(this.board.getMyBoard())){
+                    System.out.println("Game over.");
+                    return;
+                }
+        }
+    }
 
+    public void takeMovesFromArtificialPlayer(){
+        int[] xYtable = artificialPlayer.chooseCoordinates();
+        int x = xYtable[0];
+        int y = xYtable[1];
+        board.markField(this.artificialPlayer, x, y);
+        System.out.println("Artificial player tooks " + x + "," + y);
+    }
 
-//    public void takeMovesFromPlayers(){
-//        String input = player.chooseField();
-//        int[] xYtable = player.splitParseMove(input);
-//        int x = xYtable[0];
-//        int y = xYtable[1];
-//        board.takeField()
-//    }
+    public void takeMoveFromPlayer(){
+
+        boolean isChosenField = false;
+        int[] xYtable = null;
+        do {
+            do { xYtable = player.chooseCoordinates(); }
+            while(xYtable.length == 0);
```

Comment:
Jakbyś dodała jakieś wcięcia do tego do-while'a to by było o wiele bardziej czytelne.